ui/src/organizations/actions/orgs.ts

```diff
@@ -168,10 +186,13 @@ export const createOrgWithBucket = (
     dispatch(addOrg(createdOrg))
     dispatch(push(`/orgs/${createdOrg.id}`))
 
-    await client.buckets.create({
-      ...bucket,
-      orgID: createdOrg.id,
-    })
+    bucket.orgID = createdOrg.id
```

Comment:
would it be possible to remove this line and instead do something like

```
const bucketResp = await api.postBucket({data: {...bucket, orgID: createdOrg.id}})
``` 

right below? As a convention we pretty much never mutate POJOs after they are created, since it tends to cause some subtle bugs.

The type checker might complain, in which case I think a type assertion would probably work:

```
const data = {...bucket, orgID: createdOrg.id} as Bucket

const bucketResp = await api.postBucket({data})
```

Reply:
Awesome. Thats definitely good feedback. In retrospect yeah, I totally shouldn't have mutated bucket. I've got to get back into the FP style of always just make a new thing.